packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -219,7 +224,11 @@ export async function load_node({
 								/** @type {import('types/helper').ResponseHeaders} */
 								const headers = {};
 								for (const [key, value] of response.headers) {
-									if (key !== 'etag' && key !== 'set-cookie') headers[key] = value;
+									if (key === 'set-cookie') {
+										set_cookie_headers = set_cookie_headers.concat(value);
```

Comment:
The cookie value may included multiple k/v entries with comma joined see https://datatracker.ietf.org/doc/html/rfc2616#section-4.2. We should convert them to array of string. Here is a implementation: https://github.com/nfriedly/set-cookie-parser/blob/9cb7f099b9328bc6ab7b7c1eab31d8cfb41b2e44/lib/set-cookie.js#L126